jsplot/src/main/java/tech/tablesaw/plotly/display/Browser.java

```diff
@@ -16,9 +16,15 @@ public static void main(String[] args) throws Exception {
 
   public void browse(File file) throws IOException {
     if (Desktop.isDesktopSupported()) {
-
-      Desktop.getDesktop().browse(file.toURI());
-    } else {
+      if(Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)){
```

Comment:
the formatting here is horrendous

Reply:
Do you want me to format it? I think my IDE dose double tab. Any formatting style should I follow?

Reply:
Code is formatted.

Reply:
please don't change the formatting from 2 spaces to 4 spaces